resources/service-catalog-addons/charts/service-binding-usage-controller/templates/metrics.yaml

```diff
@@ -0,0 +1,50 @@
+# Required because the Binding Usage Controller has Istio sidecar but Prometheus Operator doesn't
+apiVersion: authentication.istio.io/v1alpha1
+kind: Policy
+metadata:
+  name: {{ template "fullname" . }}
+spec:
+  peers:
+  - mtls:
+      mode: PERMISSIVE
+  targets:
+  - name: {{ template "fullname" . }}
+---
+# Dedicated Service for metrics endpoint
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  ports:
+  - name: metrics
+    port: 8080
```

Comment:
shouldn't we use {{ .Values.service.internalPort }} ?

Reply:
agree, fixed